### PR TITLE
docs: update environment-variables.md

### LINKED ISSUE #493: Add documentation about database environment variables
## Description

As a developer,
I need to understand the environment variables I need to set,
so that I can successfully run and test the backend API.

Environment Variables markdown file is missing information regarding the Database environment variables that you have to set to connect the backend to a remote database. There are at least 5 
You may have to mention that these are used only when the environment configuration is set to `dev` or `prod`.

For each mention an example of the value and the description of the variable.

The Environment Variables document is located under `/docs` folder.

### Update [Required]
- [ ] Add documentation about Database environment variables

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
1 hours

### PR DESCRIPTION
**Pull request by divyanshugit**
### Description

To run the mentoring back end in ' dev ' or  ' prod ' mode one have to export database variables as well. I have documented database environment variables in this file

Fixes #493

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: docs/environment-variables.md [hunk 1]
```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
...
+ ## Database environment variables description
+
+
+| Environment Variable | Description                                                                 | Example           |
+|----------------------|-----------------------------------------------------------------------------|-------------------|
+| DB_TYPE | Type of database you want to use | postgres, mysql |
+| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123, jhonDoe | 
+| DB_PASSWORD | Database password of the `DB_USERNAME` user | mypwd123 |
+| DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db |
+| DB_NAME | Name of the database to which you want to connect| Users.db, Tasks.db |
```
Comment (isabelcosta) [inline docs/environment-variables.md]: these variable examples are not correct since it is not about the table name, but the database name. So for example, if you use a service like ElephantSQL for example. So as examples maybe you could have `database1`? It's hard to come up with an easy to understand example 🤔

File: docs/environment-variables.md [hunk 2]
```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
...
+export DB_ENDPOINT=<database_endpoint>
+export DB_NAME=<database_name> |
+```
+ ## Database environment variables description
+
+
+| Environment Variable | Description                                                                 | Example           |
+|----------------------|-----------------------------------------------------------------------------|-------------------|
+| DB_TYPE | Type of database you want to use | postgres, mysql |
+| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123, jhonDoe |
```
Comment (isabelcosta) [inline docs/environment-variables.md]:
```suggestion
| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123 | 
```

Just one example here is ok :) so that it does not confuse contributors into thinking they have to put 2 values

File: docs/environment-variables.md [hunk 3]
```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
...
+# Database Enviorment Variables
+
+To run the backend in other than `local` mode i.e `prod` or `dev` you need to export database environment variables.
+These are the required database environment variables.
+```
+export DB_TYPE=<database_type>
+export DB_USERNAME=<database_username>
+export DB_PASSWORD=<database_password>
+export DB_ENDPOINT=<database_endpoint>
+export DB_NAME=<database_name> |
```
Comment (isabelcosta) [inline docs/environment-variables.md]:
```suggestion
export DB_NAME=<database_name>
```

File: docs/environment-variables.md [hunk 4]
```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
...
+```
+ ## Database environment variables description
+
+
+| Environment Variable | Description                                                                 | Example           |
+|----------------------|-----------------------------------------------------------------------------|-------------------|
+| DB_TYPE | Type of database you want to use | postgres, mysql |
+| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123, jhonDoe | 
+| DB_PASSWORD | Database password of the `DB_USERNAME` user | mypwd123 |
+| DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db |
```
Comment (isabelcosta) [inline docs/environment-variables.md]:
```suggestion
| DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db, remote-database-url:5432 |
```

This is to give an idea that they can use a remote URL

File: docs/environment-variables.md [hunk 5]
```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
 export FLASK_ENVIRONMENT_CONFIG=dev
 ```
 
-Windows user should use **set** instead of **export** for setting these environment variables. <br/>
```
Comment (isabelcosta) [inline docs/environment-variables.md:73]: No need to remove this, this line in intentional to help Windows users. Can you undo this change, please?

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@divyanshugit this PR is almost done :) Thank you for your contribution here 🎉

File: docs/environment-variables.md [hunk 6]
```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
...
-
 - Content of `.env` file:
-
 ```
 FLASK_ENVIRONMENT_CONFIG=dev
 SECRET_KEY='some_random_key'
 (...)
 ```
+
+# Database Enviorment Variables
```
Comment (isabelcosta) [inline docs/environment-variables.md]:
```suggestion
# Database Environment Variables
```

File: docs/environment-variables.md [hunk 7]
```diff
@@ -69,14 +69,35 @@ export KEY=VALUE
...
+ ## Database environment variables description
+
+
+| Environment Variable | Description                                                                 | Example           |
+|----------------------|-----------------------------------------------------------------------------|-------------------|
+| DB_TYPE | Type of database you want to use | postgres, mysql |
+| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123 | 
+| DB_PASSWORD | Database password of the `DB_USERNAME` user | mypwd123 |
+| DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db, remote-database-url:5432 |
+| DB_NAME | Name of the database to which you want to connect| Users.db, Tasks.db |
```
Comment (isabelcosta) [inline docs/environment-variables.md]:
```suggestion
| DB_NAME | Name of the database to which you want to connect | mentorship |
```

Comment (isabelcosta) [reply docs/environment-variables.md]: @divyanshugit, I gave you an example here which can be less misleading :) Please ping after you fix this little change and then I can approve

**Comment by divyanshugit** [comment]
@isabelcosta I have made the changes like you said. Just review once !

**Review by isabelcosta — APPROVED** [review]
Great work @divyanshugit 👏
Thank you so much for your contribution!!

**Review by gaurivn — APPROVED** [review]

**Review by gaurivn — APPROVED** [review]
LGTM, Thank you

**Comment by isabelcosta** [comment]
Thank you @gaurivn 🙌  for your review!

**Comment by isabelcosta** [comment]
Your first PR was merged on mentorship Backend 👏 @divyanshugit
